streamingcli/jupyter/jar_handler.py

```diff
@@ -23,6 +24,23 @@ def create_project_jars(self) -> None:
         elif not os.path.isdir(self.project_jars):
             raise ValueError(f'{self.project_jars} is already in the project directory with incompatible type.')
 
+    def get_jars_using_plugin(self) -> str:
```

Comment:
I would see something like `get_names_of_jars_using_plugin` or `get_classpaths_of_jars_using_plugin` instead. Right now the name suggests getting some kind of a wrapper around jars, not their classpaths.
Especially considering that in `build_command.py` the result is called `plugin_jars_paths` anyway.